src/plotting/plot.component.test.tsx

```diff
@@ -1,5 +1,65 @@
 import React from 'react';
 import { render, fireEvent } from '@testing-library/react';
+import { FullScalarChannelMetadata, Record, ScalarChannel } from '../app.types';
+import ConnectedPlot, {
+  ConnectedPlotProps,
+  getFormattedAxisData,
+  formatTooltipLabel,
+} from './plot.component';
+import {
+  testRecords,
+  renderComponentWithProviders,
+  generateRecord,
+  testChannels,
+} from '../setupTests';
+
+const testScalarChannels = testChannels as FullScalarChannelMetadata[];
+
+jest.mock('victory', () => {
```

Comment:
Now that we're mocking victory at the top - can we remove the `doMock` call in the `Plot` tests?

Reply:
That would make sense but are we still able to verify what props are being passed to mocked components without making use of snapshots? Or we could just use snapshots across the whole file if you prefer

Reply:
Ah, yeah the problem is the mock functions cause issues when done in the `jest.mock` due to them being undefined. If this works it works, it's just a little more repetitive. Unless we have a `describe` block that wraps around the whole test suite and do the `doMock` in there that's usable by both components' tests?

Reply:
I might be able to get that working, I admit I'm a little unhappy with the current method too

Reply:
@louise-davies would you mind looking over my last commit which fixes this issue? Then I believe I'm happy for this to be merged if you are